modules/web/src/app/core/services/cluster.ts

```diff
@@ -282,8 +286,16 @@ export class ClusterService {
   }
 
   health(projectID: string, clusterID: string): Observable<Health> {
-    const url = `${this._newRestRoot}/projects/${projectID}/clusters/${clusterID}/health`;
-    return this._http.get<Health>(url).pipe(catchError(() => of<Health>({} as Health)));
+    const mapKey = projectID + '-' + clusterID;
```

Comment:
I think we should either use `shareReplay` or revert these changes because this method is not called from multiple places on a single view. And there is no benefit in storing previous health value in a map because pending icon is displayed while health is loading which is fine.